src/frontend/org/voltdb/utils/CSVLoader.java

```
@@ -574,10 +574,18 @@ private static void configuration() {
      */
     public static Client getClient(ClientConfig config, String[] servers,
             int port) throws Exception {
+        config.setTopologyChangeAware(true); // Set client to be topology-aware
         final Client client = ClientFactory.createClient(config);
-
         for (String server : servers) {
-            client.createConnection(server.trim(), port);
+            // Try connecting servers one by one until we have a success
+            try {
+                client.createConnection(server.trim(), port);
+                break;
+            } catch(Exception e) { /* Do Nothing */ }
```

Right now you're swallowing all exceptions, which may hide problems that have nothing to do with servers being unavailable. Perhaps check for IOException and UnknownHostException, since createConnection() declares that it can throw them?